data/abilities.ts

```diff
@@ -2767,11 +2767,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 		num: 232,
 	},
 	propellertail: {
-		onModifyMove(move) {
```

Comment:
You need to make sure the new Stalwart code is in Propeller Tail too.

Reply:
oh right durrr
my bad